content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -92,6 +95,9 @@ A _dependency entry_ (`deps`) can have these fields:
   _checksum_ for HDFS and WebHDFS. See `dvc import-url` for more information.
 - `size`: Size of the file or directory (sum of all files).
 - `nfiles`: If a directory, number of files inside.
+- `isexec`: Whether this is an executable file. DVC preserves execute
```

Comment:
hmm, sorry guys .. I'm way too late (and approving this PR) ... just to clarify - what does it mean for _deps_. I though pull/checkout do now do anything to deps?

How would it work if out say exec:True and dep says exec:False?

Reply:
Oops, this was not meant to be here. Deleting.